app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -162,22 +178,46 @@
     private Realm realm = Realm.getDefaultInstance();
     private String caption;
     private boolean atleastOneShare = false;
+
+    /**
+     * Official colors of accounts logo
+     */
     private int[] cellcolors_LightTheme = {R.color.facebook_color, R.color.twitter_color,
             R.color.instagram_color, R.color.wordpress_color,
             R.color.pinterest_color, R.color.flickr_color,
             R.color.nextcloud_color, R.color.imgur_color, R.color.dropbox_color,
-            R.color.googlePlus_color, R.color.accent_black, R.color.other_share_color};
-
+            R.color.googlePlus_color, R.color.accent_black, R.color.owncloud_color
+            , R.color.other_share_color};
     private AccountPresenter accountPresenter;
-    private int[] cellcolors_DarkTheme = {R.color.facebook_color_darktheme, R.color.twitter_color_darktheme, R.color.instagram_color_darktheme,
-            R.color.wordpress_color_darktheme, R.color.pinterest_color_darktheme, R.color.flickr_color_darktheme, R.color.nextcloud_color_darktheme, R.color.imgur_color_darktheme, R.color.dropbox_color_darktheme, R.color.other_share_color_darktheme, R.color.other_share_color_darktheme};
+
+    /**
+     * Collection of dark theme colors for accounts table in sharing activity
+     */
+    private int[] cellcolors_DarkTheme = {R.color.facebook_color_darktheme, R.color.twitter_color_darktheme
+            , R.color.instagram_color_darktheme, R.color.wordpress_color_darktheme
+            , R.color.pinterest_color_darktheme, R.color.flickr_color_darktheme
+            , R.color.nextcloud_color_darktheme, R.color.imgur_color_darktheme
+            , R.color.dropbox_color_darktheme, R.color.other_share_color_darktheme
+            , R.color.owncloud_color, R.color.other_share_color_darktheme};
```

Comment:
use a lighter shade color for dark theme like all the other colors. Donot use the same color. Use this #45556c for dark theme.